layouts/contribute/list.html

```diff
@@ -0,0 +1,99 @@
+                        <div class="print-label"></div>
+
+                        <h2 class="custom-margin">Sign CLA Electronically</h2>
+                        <p>Alternatively to sending by postal mail, email, or fax, you may also choose to sign the form electronically:</p>
+                        <br>
+
+                        <form role="form" class="form-horizontal contact" action="https://singlecell.biolab.si/contribute/" method="POST" name="license-form">
+                            <div class="form-group">
+                                <label class="col-md-2 col-sm-3 col-xs-10 span4" for="fullname">Full Name:</label>
+                                <div class="inline-label-input col-md-4 col-sm-4 col-xs-10">
+                                    <input class="form-control" type="text" id="fullname" name="Full Name" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-2 col-sm-3 col-xs-10" for="email">E-mail:</label>
+                                <div class="inline-label-input col-md-4 col-sm-4 col-xs-10">
+                                    <input class="form-control" type="email" id="email" name="E-mail" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-2 col-sm-3 col-xs-10" for="address">Mailing Address:</label>
+                                <div class="inline-label-input col-md-4 col-sm-4 col-xs-10">
+                                    <textarea class="form-control" id="address" name='Address' rows="7" required></textarea>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-2 col-sm-3 col-xs-10" for="country">Country:</label>
+                                <div class="inline-label-input col-md-4 col-sm-4 col-xs-10">
+                                    <input class="form-control" type="text" id="country" name="Country" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-2 col-sm-3 col-xs-10" for="number">Telephone Number:</label>
+                                <div class="inline-label-input col-md-4 col-sm-4 col-xs-10">
+                                    <input class="form-control" type="text" id="number" name="Number" size="25" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-md-2 col-sm-3 col-xs-10" for="signature">Electronic Signature:<br>(Type "I AGREE")</label>
```

Comment:
Could this be made in a different form?
a) Have to pre-fill the value with I AGREE so the user doesn't have to type it herself?
b) Have a checkbox instead?
Just to make it easier.

Reply:
I will add a checkbox.